app/modules/intelligence/agents/multi_agent_config.py

```
@@ -0,0 +1,114 @@
+        == "true",
+        "codebase_qna_agent": os.getenv("QNA_MULTI_AGENT", "true").lower() == "true",
+        "debugging_agent": os.getenv("DEBUG_MULTI_AGENT", "true").lower() == "true",
+        "unit_test_agent": os.getenv("UNIT_TEST_MULTI_AGENT", "true").lower() == "true",
+        "integration_test_agent": os.getenv(
+            "INTEGRATION_TEST_MULTI_AGENT", "true"
+        ).lower()
+        == "true",
+        "LLD_agent": os.getenv("LLD_MULTI_AGENT", "true").lower() == "true",
+        "code_changes_agent": os.getenv("CODE_CHANGES_MULTI_AGENT", "true").lower()
+        == "true",
+    }
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)